containerized_ml_models/api/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM continuumio/miniconda3
+
+COPY api.py model.pkl requirements.txt ./
+
+RUN conda create --name virtual_env 
+RUN conda run -n virtual_env pip install -r requirements.txt
```

Comment:
You can chain RUN commands with '&& \':
`RUN conda create  --name virtual_env && \`
`conda run -n virtual_env pip install -r requirements.txt`